script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -1244,25 +1254,25 @@ After all this time, somebody finally shouted 'Dawn isn't coming!'.
 -- Page 641, Offset 35112.
 -- ひとりひとり、丹念に、でも何の愛着もなく■された。
 // Assuming 殺された.
-One after another, we were all painstakingly, yet cold-heartedly, ■■■ by her.
+One after another, we were all painstakingly, yet indifferently, ■■■ by her.
 }
 [sha:2d969d41645e8d62c59911af0a8a1b9627ed83aa]{
 -- Page 642, Offset 35113.
 -- ……いちじかんにひとりぐらいのペースだったから、たぶん、みっかぐらいたったあとのはなしです。
 // i love hiragana!
-She'd been going at a pace of around one person an hour,
- so by this point it must have been around three days since she began.
+shed been going at a pace of around one person an hour
+ so by this point it must have been around three days since she began
```

Comment:
probably indicate this is where the "hiragana-kids talk" starts so proofreaders dont get confused